.github/workflows/verify-pr-commit.yml

```diff
@@ -395,7 +395,7 @@ jobs:
         uses: actions/checkout@v4
       - name: Set Env Vars
         run: |
-          echo "BIN_FILENAME=frequency.local.frequency-rococo-local.pr" >> $GITHUB_ENV
+          echo "BIN_FILENAME=frequency.local.frequency-local.pr" >> $GITHUB_ENV
```

Comment:
binary names update to reflect `testnet or local` as the binaries will be able to start either rococo or paseo based on spec provided